src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -50,6 +54,42 @@ public void generates_and_saves_pdf() throws IOException {
         document.validate();
     }
 
+    @Test
+    public void generate_and_save_pdf_and_return_same_id_on_resubmit() throws IOException {
```

Comment:
Please add exception test cases

Reply:
Exception comes from `SampleData.letter()` while reading the resource. Would be like testing a test